src/Data/MapObject.cpp

```diff
@@ -576,6 +576,18 @@ QHash<QString, QString> OsmAnd::MapObject::getResolvedAttributes() const
             result.insert(decodedAttribute.tag, decodedAttribute.value);
         }
     }
+    for (const auto& captionAttributeId : constOf(captionsOrder))
```

Comment:
Not clear why captions added in getResolvedAttributes method. Will it break other code?

Reply:
In [legacy code](https://github.com/osmandapp/OsmAnd-core-legacy/blob/144f0a0770ab448d33e2471a6dd8b58de51e67ad/native/src/java_wrap.cpp#L1292) all names convert to tags:
```
for (; it != robj->objectNames.end(); it++) {
...
    ienv->CallVoidMethod(resobj, jmethod_RenderedObject_putTag, ts, vs);
```

Reply:
See whole MapObject. captions and captionsOrder using only with name indexes:
getCaptionInNativeLanguage
getCaptionInLanguage
getCaptionsInAllLanguages